Food_Tracker_App/app/src/main/res/layout/meal_plan_day_content.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:cardView="http://schemas.android.com/apk/res-auto"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_margin="16dp"
+    android:orientation="horizontal"
+    cardView:cardBackgroundColor="@color/background_dark"
+    cardView:cardCornerRadius="20dp"
+    cardView:contentPadding="16dp">
```

Comment:
Not sure if I like the card view here because we have the nested recycler views that look as though they are cutoff, but this is just an opinion feel free to disregard this